### PR TITLE
Add EIP: Purpose bound money

### PR DESCRIPTION
**Pull request by proj-orchid-straitsx**
The  team is proposing to standardise the implementation of purpose-bound money (PBM) in an EIP. The aim is to provide a common specification for wallet players to be able to interact freely with PBM smart contracts issued by various PBM creators for various business use cases. 

What are PBMs? 
PBMs are a hybrid form of programmable payment and programmable money, which are bearer instruments, with self-contained programming logic and are transferrable between two parties without intermediaries. 

What are the practical uses? 
Purpose-bound money can be used to implement government disbursement tokens, shopping voucher tokens, prepaid tokens, rewards points token, purpose bound donation token, school allowance token etc. For additional information on PBM, please refer to:
  * [Project Orchid Whitepaper ](https://www.mas.gov.sg/-/media/mas-media-library/development/fintech/project-orchid/mas-project-orchid-report.pdf)and [media release](https://www.mas.gov.sg/schemes-and-initiatives/project-orchid)
  * [PBM Technical Whitepaper](https://www.mas.gov.sg/-/media/mas-media-library/development/fintech/pbm/pbm-technical-whitepaper.pdf) and [media release](https://www.mas.gov.sg/news/media-releases/2023/mas-proposes-standards-for-digital-money) by Monetary Authority of Singapore (Singapore’s central bank and financial supervisory authority)
  * [Live Demo Videos from Singapore Fintech Festival 2022](https://drive.google.com/drive/folders/1lF7PlRbpyrNa5VO4heQbBX0ZvDVNcgXU?usp=sharing)

What are our goals?
To develop a minimal interface that can promote interoperability between various payment market players. The proposed interface builds upon ERC-1155 to ensure widespread compatibility with existing wallets. 

Our team would appreciate any comments on the proposal’s design, feasibility as well as any security concerns. As the proposal is in the draft stage, the proposal will continue to evolve as we receive comments from the community.

[Link to discussion at Ethereum Magician](https://ethereum-magicians.org/t/eip-7291-purpose-bound-money/14973)

### CODE REVIEW

**Comment by alcedo** [comment]
image

not sure why this is now suddenly reporting an error

File: EIPS/eip-7292.md [hunk 1]
```diff
@@ -0,0 +1,636 @@
+---
+eip: 7292
```
Comment (abcoathup) [inline EIPS/eip-7292.md:2]:
```suggestion
eip: 7291
```
Assigning EIP number using PR number of first opened PR #7291
Please also update the filenames.

Comment (wongtsejian) [reply EIPS/eip-7292.md]: Thanks for your quick response. I have updated the EIP number to 7291 and changed the filenames.

**Review by wongtsejian — APPROVED** [review]

**Comment by proj-orchid-straitsx** [comment]
Hi, is there anything else that we need to do before this can be merged?

File: EIPS/eip-7291.md [hunk 2]
```diff
@@ -0,0 +1,636 @@
...
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-06-24
+requires: 165, 173, 1155
+---
+
+## Abstract
+
+This proposal outlines a smart contract interface that builds upon the [ERC-1155](./eip-1155.md) standard to introduce the concept of a purpose bound money (PBM) defined in the [Project Orchid Whitepaper](../assets/eip-7291/MAS-Project-Orchid.pdf) and [PBM Technical Whitepaper](../assets/eip-7291/PBM-Technical-Whitepaper.pdf).
```
Comment (lightclient) [inline EIPS/eip-7291.md:16]: An ERC should be able to stand on it's own. We allow linking to resources to support the ERC, but I think these PDFs are too much. PBM should be defined here and it shouldn't refer to Project Orchid at all.

**Review by lightclient — CHANGES_REQUESTED** [review]

File: EIPS/eip-7291.md (hunk 2 above)
Comment (wongtsejian) [reply EIPS/eip-7291.md]: Hi  @lightclient, we have make changes to address your comments and a comment on Eth Magician forum. Thanks.

**Review by wongtsejian — APPROVED** [review]
* added detailed definition of PBMs
* removed reference to project orchid articles

**Comment by wongtsejian** [comment]
Hi @lightclient ,

Gentle reminder, could you check if our changes have addressed your request?

**Comment by alcedo** [comment]
hi @lightclient any feedbacks on this for the next steps?

**Comment by alcedo** [comment]
@abcoathup would you be able to advise on the next step?

**Comment by abcoathup** [comment]
> @abcoathup would you be able to advise on the next step?

@alcedo You need an EIP editor to review (which I am not) to get this merged as a draft ERC.

There is an EIP office hours tomorrow that you might want to join to ask about getting reviewed and anything outstanding.
https://github.com/ethereum-cat-herders/EIPIP/issues/262

**Comment by alcedo** [comment]
thanks @abcoathup !

**Comment by SamWilsn** [comment]
I am closing this pull request because we are in the process of separating EIPs and ERCs into distinct repositories. Unfortunately, as far as we are aware, GitHub does not provide any tools to ease this migration, so every pull request will need to be re-opened manually.

As this is a PR to create / modify an ERC, I will kindly ask you to redirect this to the new repository at [ethereum/ERCs](https://github.com/ethereum/ERCs). **We have prepared [a guide](https://github.com/ethereum/ERCs/issues/8) to help with the process.**

If there is relevant history here, please link to this PR from the new pull request.

On behalf of the EIP Editors, I apologize for this inconvenience.

**Pull request closed without merging**